include/Print.h

```diff
@@ -93,7 +93,7 @@ class __print_holder {
     std::ostream* m_out = &std::cout; // Cannot use refernce
                                      // becouse it may be replaced
     std::string_view m_sep = " ";
-    std::string_view m_end = " ";
+    std::string_view m_end = "\n";
```

Comment:
This end of line is platform specific.

Reply:
I haven't found proper way to use platform specific endline character(s).

std::endl prints '\n' too.